client/components/List/index.js

```diff
@@ -115,9 +115,12 @@ class List extends Component {
      };
 
     selectCategory = (categoryId, todo) => {
+      categoryId = parseInt(categoryId)
```

Comment:
Why did you add that ? You never use that.

Reply:
categoryId = parseInt(categoryId)

      const todos = this.props.todos;
      const oldCategoryId = todo.categoryId;
      todo.categoryId = categoryId;

Reply:
this was the bug I fixed, if I won't parseInt then it was string and later there was problems with it as categoryId in todos isn't string...